types/generated/snowbound/core/ApiError.ts

```diff
@@ -0,0 +1,24 @@
+/* istanbul ignore file */
```

Comment:
super-nit: not sure we _need_ another level of directory nesting for `generated` - are we going to co-locate generated and non-generated types for one API?

Reply:
I don't know, maybe? I think it's useful to have a clear delineation between "things you should edit" and "things you shouldn't".

Reply:
I see. Our GraphQL generated stuff is in the wrong place then - I am used to CI that will re-gen and ensure that things are up-to-date, as well. In order for CI to not trigger diffs, I'd check in the OpenAPI schema (like we did for GraphQL) and validate that the schema -> code generation is correct at merge-time.